xml/admin_rbd.xml

```diff
@@ -870,7 +892,7 @@ data blocks changed from 2097152 to 2560000</screen>
     <para>
      To enable mirroring on a pool, specify the <command>mirror pool
      enable</command> subcommand, the pool name, and the mirroring mode. The
-     mirroring mode can either be pool or image:
+     mirroring mode can either be pool or image. For example:
```

Comment:
nit: Why 'For example' is added here. The sentence says that two modes are available and below these modes are described. The previous version looked ok to me.